controllers/argocd/status.go

```diff
@@ -212,3 +216,40 @@ func (r *ReconcileArgoCD) reconcileStatusServer(cr *argoprojv1a1.ArgoCD) error {
 	}
 	return nil
 }
+
+// reconcileStatusURL will ensure that the URL status is updated for the given ArgoCD.
+func (r *ReconcileArgoCD) reconcileStatusURL(cr *argoprojv1a1.ArgoCD) error {
+
+	if cr.Spec.Server.Route.Enabled {
+		route := newRouteWithSuffix("server", cr)
+		if argoutil.IsObjectFound(r.Client, cr.Namespace, route.Name, route) {
+			if route.Spec.Host == "" {
```

Comment:
any reason for choosing `route.spec.Host` instead of `route.status.ingress.conditions.host` ?